### DISCUSSION TITLE
What exactly is the "new chat menu"?

### DISCUSSION THREAD

**Discussion by NMBlanco** [rodit/SnapMod#150] [Q&A/Support] [answered]
I enabled it with the latest SnapMod and Snapchat 11.79.0.34 but can't see any difference from the old context menu? Or is the new chat menu something that Snap Inc. enables server-sided for randomly selected users?

**Comment by NMBlanco** [comment, marked as answer]
Nevermind 😁

**Reply by RSR1337** [reply]
Its like how is snapchat on iOS + SnapMid menu
